### PR TITLE
Update UnmappedBamToAlignedBam.wdl

### PR DESCRIPTION
**Pull request by ekiernan**
Testing Changed MarkDuplicates task to use Float size(Array[File], [String])

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by nikellepetrillo — APPROVED** [review]

File: tasks/broad/UnmappedBamToAlignedBam.wdl [hunk 1]
```diff
@@ -137,16 +137,9 @@ workflow UnmappedBamToAlignedBam {
     }
   }
 
-  # Sum the read group bam sizes to approximate the aggregated bam size
-  call Utils.SumFloats as SumFloats {
```
Comment (kayleemathews) [inline tasks/broad/UnmappedBamToAlignedBam.wdl:141]: I think this task is mentioned in the [WGS Tasks and Tools table](https://broadinstitute.github.io/warp/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/README/#workflow-tasks-and-tools). Would you mind updating that?

File: pipelines/broad/dna_seq/germline/single_sample/exome/ExomeGermlineSingleSample.changelog.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 3.1.13
```
Comment (kayleemathews) [inline pipelines/broad/dna_seq/germline/single_sample/exome/ExomeGermlineSingleSample.changelog.md:1]: We should update the version number in the [Exome Overview doc](https://broadinstitute.github.io/warp/docs/Pipelines/Exome_Germline_Single_Sample_Pipeline/README).

File: pipelines/broad/dna_seq/germline/single_sample/wgs/WholeGenomeGermlineSingleSample.changelog.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+# 3.1.14
```
Comment (kayleemathews) [inline pipelines/broad/dna_seq/germline/single_sample/wgs/WholeGenomeGermlineSingleSample.changelog.md:1]: We should update the version number in the [WGS Overview doc](https://broadinstitute.github.io/warp/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/README).

**Review by kayleemathews — COMMENTED** [review]
A couple minor doc things should be updated. If you want me to make those changes let me know.

**Review by jessicaway — APPROVED** [review]
Thanks @ekiernan!

File: tasks/broad/UnmappedBamToAlignedBam.wdl (hunk 1 above)
Comment (ekiernan) [reply tasks/broad/UnmappedBamToAlignedBam.wdl:141]: Mischief managed! 

Thank you!

**Review by kayleemathews — APPROVED** [review]
LGTM! Thanks Liz!

**Pull request merged**